pulp_container/app/serializers.py

```diff
@@ -307,9 +307,11 @@ class ContainerPullThroughRemoteSerializer(RemoteSerializer):
     """
 
     policy = serializers.ChoiceField(choices=[Remote.ON_DEMAND], default=Remote.ON_DEMAND)
+    includes = serializers.JSONField(required=False, allow_null=True)
+    excludes = serializers.JSONField(required=False, allow_null=True)
```

Comment:
Maybe, we should go with `serializers.ListField` and `fields.ArrayField` like we did for `include_tags`/`exclude_tags`. This will force users to use a valid list as an input.